lib/msal-common/test/account/AuthToken.spec.ts

```diff
@@ -47,6 +47,14 @@ describe("AuthToken.ts Class Unit Tests", () => {
                         return input;
                 }
             },
+            base64UrlEncode(input: string): string {
+                switch (input) {
+                    case '{kid: "XnsuAvttTPp0nn1K_YMLePLDbp7syCKhNHt7HjYHJYc"}':
```

Comment:
This should be:
```suggestion
                    case '{ "kid" : "XnsuAvttTPp0nn1K_YMLePLDbp7syCKhNHt7HjYHJYc"}':
```

I'd suggest building the object above and JSON.stringify-ing it, then passing it to the case clause

Reply:
Addressed